Tasks/AzureRmWebAppDeployment/Utility.ps1

```diff
@@ -136,6 +136,31 @@ function Get-MsDeployCmdArgs
         $msDeployCmdArgs += ( " " + $AdditionalArguments)
     }
 
+    $collectionUri = Get-VstsTaskVariable -Name System.TeamFoundationCollectionUri -Require
```

Comment:
@pragupta-ms - Can we extract out this to a method which will return $userAgent , and in this method just append $userAgent to $msDeployCmdArgs